provider/openstack/provider.go

```diff
@@ -1086,7 +1086,7 @@ func (e *Environ) StartInstance(args environs.StartInstanceParams) (*environs.St
 				return nil
 			},
 			NotifyFunc: func(lastError error, attempt int) {
-				args.StatusCallback(status.Provisioning, fmt.Sprintf("%q, wait 10 seconds before retry, attempt %d", lastError, attempt), nil)
+				args.StatusCallback(status.Provisioning, fmt.Sprintf("%s, wait 10 seconds before retry, attempt %d", lastError, attempt), nil)
```

Comment:
I think its fine not to %q both of them, but I think its usually useful to %q instance ids. We could go without any quoting if you really think it looks nicer, but I often like "foo" around variables that are being inserted. Especially since instance-id is foreign data. If we kept the 'errStillBuilding' using %q we would have:
instance "db6b89bf-04d4-4859-ab2c-5796225af79f" has status BUILD, wait 10 seconds before retry, attempt 3

which I think is reasonable.